cmd/intelpcr/cmd.go

```diff
@@ -0,0 +1,66 @@
+package main
```

Comment:
doesn't build due to
../../pkg/intel/metadata/manifest/key/manifest.go:38:16: undefined: manifest.TPMAlgorithm